assets/sass/patterns/molecules/about-profile.scss

```diff
@@ -19,18 +41,22 @@
   @include margin(12 0 12 0);
   max-width: 100%;
   width: 100%;
-}
 
-.about-profile__image {
-  width: 100%;
-}
-
-@media only all and (min-width: 700px) {
-
-  .about-profile__picture {
+  @media only all and (min-width: 700px) {
     float: right;
     width: 250px;
     @include margin(24, "left");
   }
 
+  @media only all and (min-width: #{$bkpt-site--medium}px) {
+    .about-profiles__item .about-profile--popup & {
```

Comment:
As far as I understand this hides the element `.about-profile--popup` content embedded in the page, as long as we are in the "large page" scenario. Then we clone that element when the popup needs to be opened, putting the clone in a special place in the page. But then the content ends up being present twice in the page, which is strange.